cognite_toolkit/_cdf_tk/loaders/_resource_loaders/transformation_loaders.py

```diff
@@ -377,6 +411,31 @@ def delete(self, ids: SequenceNotStr[str | int]) -> int:
             self.client.transformations.delete(id=existing, ignore_unknown_ids=True)
         return len(existing)
 
+    def _update_nonce(self, items: Sequence[TransformationWrite]) -> None:
+        for item in items:
+            if item.external_id:
```

Comment:
is it possible to not have external_id? If internal id case is supported some more code is needed, if not it should probably be a check&error rather than "if"?

Reply:
No, see line 169. Toolkit requires externalId on transformations.